test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -2408,48 +2415,182 @@ describe('DefaultDeviceController', () => {
 
   describe('preview', () => {
     let element: HTMLVideoElement;
+    let spy1: sinon.SinonSpy;
+    let spy2: sinon.SinonSpy;
+    let spy3: sinon.SinonSpy;
+    let gUM: sinon.SinonSpy;
 
     beforeEach(() => {
       const videoElementFactory = new NoOpVideoElementFactory();
       element = videoElementFactory.create();
+      spy1 = sinon.spy(deviceController, 'releaseMediaStream');
+      spy2 = sinon.spy(DefaultVideoTile, 'disconnectVideoStreamFromVideoElement');
+      spy3 = sinon.spy(DefaultVideoTile, 'connectVideoStreamToVideoElement');
+      gUM = sinon.spy(navigator.mediaDevices, 'getUserMedia');
     });
 
-    it('connects the video stream to the video element', async () => {
-      const spy1 = sinon.spy(DefaultVideoTile, 'disconnectVideoStreamFromVideoElement');
-      const spy2 = sinon.spy(DefaultVideoTile, 'connectVideoStreamToVideoElement');
-      const gUM = sinon.spy(navigator.mediaDevices, 'getUserMedia');
+    afterEach(() => {
+      spy1.restore();
+      spy2.restore();
+      spy3.restore();
+      gUM.restore();
+    });
+
+    it('does not disconnect or connect the video stream for preview if no active video exists', () => {
+      deviceController.startVideoPreviewForVideoInput(element);
+
+      expect(spy1.called).to.be.false;
+      expect(spy2.called).to.be.false;
+      expect(spy3.called).to.be.false;
+      expect(gUM.called).to.be.false;
+    });
+
+    it('connects the transform video stream to the video element if there is a transform device stream', async () => {
+      // It is important that the domMockBehavior setup is before the original video stream generation to mock browser behavior.
+      domMockBehavior.createElementCaptureStream = new MediaStream();
```

Comment:
nit: Can we create the stream add the video track before setting it to domMockBehavior.